src/Cronofy/AddToCalendarRequestBuilder.cs

```diff
@@ -130,14 +141,98 @@ public AddToCalendarRequestBuilder UpsertEventRequest(UpsertEventRequest upsertE
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/> 
+        /// </returns>
+        public AddToCalendarRequestBuilder AvailabilityRequestBuilder(IBuilder<AvailabilityRequest> availabilityRequestBuilder)
+        {
+            Preconditions.NotNull("availabilityRequestBuilder", availabilityRequestBuilder);
+
+            this.availabilityRequestBuilder = availabilityRequestBuilder;
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the availability details of the request.
+        /// </summary>
+        /// <param name="availabilityRequest">
+        /// The event details for the request, must not be null.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/>.
+        /// </returns>
+        public AddToCalendarRequestBuilder AvailabilityRequest(AvailabilityRequest availabilityRequest)
+        {
+            Preconditions.NotNull("availability", availabilityRequest);
+
+            this.availabilityRequestBuilder = Builder.Wrap(availabilityRequest);
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the target calendar details of the request.
+        /// </summary>
+        /// <param name="sub">
+        /// The sub for the target calendar.
+        /// </param>
+        /// <param name="calendarId">
+        /// The target calendar's id.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/>.
+        /// </returns>
+        public AddToCalendarRequestBuilder TargetCalendars(string sub, string calendarId)
+        {
+            Preconditions.NotBlank("sub", sub);
+            Preconditions.NotBlank("calendarId", calendarId);
+
+            return TargetCalendars(new[] { new AddToCalendarRequest.TargetCalendar { 
+                    Sub = sub, 
+                    CalendarId = calendarId 
+                } });
+        }
+
+        /// <summary>
+        /// Sets the target calendar details of the request.
+        /// </summary>
+        /// <param name="targetCalendars">
+        /// The target calendars.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/>.
+        /// </returns>
+        public AddToCalendarRequestBuilder TargetCalendars(IEnumerable<AddToCalendarRequest.TargetCalendar> targetCalendars)
+        {
+            this.targetCalendars = targetCalendars;
+
+            return this;
+        }
+        
+
         /// <inheritdoc />
         public AddToCalendarRequest Build()
         {
-            return new AddToCalendarRequest
+            var request = new AddToCalendarRequest
             {
                 OAuth = this.oauthBuilder.Build(),
                 Event = this.upsertEventRequestBuilder.Build(),
+                TargetCalendars = this.targetCalendars
```

Comment:
We've generally adopted the trailing comma so diffs are cleaner when things are added/removed